lib/atlassian-jwt-authentication/verify.rb

```diff
@@ -1,16 +1,29 @@
 require 'jwt'
 
 module AtlassianJwtAuthentication
-  class Verify
+  class JWTVerification
+    attr_accessor :addon_key, :jwt, :request, :exclude_qsh_params, :logger
 
-    def self.verify_jwt(addon_key, jwt, request, exclude_qsh_params = [])
+    def initialize(addon_key, jwt, request, &block)
```

Comment:
I thought of this class as a simple service class and I'm puzzled with this change. Hey, could you tell me more what's the intention here? 

I think you didn't want to add a next argument (logger) to verify and decided to split everything into parts, but this class has only one method and one purpose, why make it instantiable?

Reply:
@pawelniewie Yeah, I needed to add one more optional `logger` parameter to `verify_jwt`. We already had the `exclude_qsh_params` param which was optional. These 2 are now set to their default values in the constructor. The new style also reduces the amount of parameters needed to be sent to the `log` helper in that class. If I keep it a method instead of an instance, I have to pass the `logger` argument down to this helper as well. I felt that turning this into an instantiable class makes it more extensible for the future.